.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
```

Comment:
See my comment in #41 about this, though this will get updated once #41 is merged of course